doc/langref.html.in

```diff
@@ -4740,6 +4740,43 @@ fn cmpxchgWeakButNotAtomic(comptime T: type, ptr: *T, expected_value: T, new_val
       </p>
       {#see_also|@cVaArg|@cVaCopy|@cVaEnd#}
       {#header_close#}
+	
+      {#header_open|@deprecated#}
+      <pre>{#syntax#}@deprecated(value: anytype) @TypeOf(value){#endsyntax#}</pre>
+      <pre>{#syntax#}@deprecated() void{#endsyntax#}</pre>
+      <p>
+	  Used to mark a given code path as deprecated. It evaluates to the same value
+	  passed in as argument, or the {#syntax#}void{#endsyntax#} value when given none.
+      </p>
+      <p>
+      As an example, in Zig 0.14.0 {#syntax#}std.time.sleep{#endsyntax#} was
+	  deprecated and the sleep function was moved to {#syntax#}std.Thread.sleep{#endsyntax#}.
+	  This is how this deprecation could have been expressed:
+	   
+      {#syntax_block|zig|lib/std/time.zig#}
+pub const sleep = @deprecated(std.Thread.sleep); // moved
+      {#end_syntax_block#}
```

Comment:
The langref ought not to refer to specific old deprecations IMO. Perhaps:
```suggestion
      For instance, consider a function which is moved to a different namespace, with a deprecated alias in the old namespace for compatibility. This is written as follows:
	   
      {#syntax_block|zig|old_namespace.zig#}
pub const my_function = @deprecated(new_namespace.my_function); // moved
      {#end_syntax_block#}
```

Reply:
Also, is it correct for syntax blocks to be inside a `<p>`? IIRC they should be outside of it, but I haven't checked.

Reply:
Apparently `figure` cannot go under `p` (but `pre` can). Moved it out since the codeblocks are enclosed in a `figure` element.

Reply:
I've used a slightly different example than the one you provided here, but kept the same general idea.